scripts/cam/plots/exevs_href_grid2obs_cape_plots.sh

```diff
@@ -0,0 +1,294 @@
+
+         chmod +x  run_py.${stats}.${thresh}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${valid_time}.sh
+
+         echo "${DATA}/run_py.${stats}.${thresh}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${valid_time}.sh" >> run_${stats}.${thresh}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${valid_time}.sh
+
+         chmod +x  run_${stats}.${thresh}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${valid_time}.sh 
+         echo "${DATA}/run_${stats}.${thresh}.${score_type}.${lead}.${VAR}.${dom}.${FCST_LEVEL_value}.${valid_time}.sh" >> run_all_poe.sh
+
+
+     done #end of FCST_LEVEL_value
+
+    done #end of VAR
+
+  done #end of fcst_lead
+
+ done #end of dom 
+
+ done #end of score_type
+
+ done #end of thresh 
+
+done #end of stats 
+
+done #end of valid_time
+
+chmod +x run_all_poe.sh
+
+
+if [ $run_mpi = yes ] ; then
+  export LD_LIBRARY_PATH=/apps/dev/pmi-fix:$LD_LIBRARY_PATH
+   mpiexec -np 768 -ppn 84 --cpu-bind verbose,depth cfp ${DATA}/run_all_poe.sh
+else
+  ${DATA}/run_all_poe.sh
+fi
+
+
+cd $plot_dir
+ 	
+for score_type in lead_average threshold_average; do
+
+ for valid in 00z 12z ; do
+
+  for var in cape mlcape ; do
+   if [ $var = cape ] ; then
+       level=l0
+       stats="csi fbias"
+   elif [ $var = mlcape ] ; then
+       level=ml
+       stats="csi fbias"
+   fi
+
+  if [ $score_type = lead_average ] ; then
+     scoretype=fhrmean
+     leads="all"
+  else 
+     scoretype=threshmean
+     leads="f6 f12 f18 f24 f30 f36 f42 f48"
+  fi 
+
+  for stat in $stats ; do
+
+   for domain in conus conus_east conus_west conus_south conus_central alaska appalachia cplains deepsouth greatbasin greatlakes mezquital midatlantic northatlantic nolains nrockies pacificnw pacificsw prairie southeast southwest splains nplains srockies ; do
+
+    for lead in $leads ; do   
+      if [ $lead = f6 ] ; then
+	  new_lead=f06
+      else 
+	  new_lead=$lead
+      fi
+
+     if [ $domain = alaska ] ; then
+         new_domain=$domain
+     else
+         new_domain=buk_${domain}
+     fi
+
+
+     if [ $score_type = lead_average ] ; then
+   
+          for thresh in ge250 ge500 ge1000 ge2000 ; do
+           mv ${score_type}_regional_${domain}_valid_${valid}_${var}_${stat}_${thresh}.png evs.href.${stat}.${var}_${level}.${thresh}.last${past_days}days.${scoretype}_valid_${valid}.${new_domain}.png
+          done
+     else
+          mv ${score_type}_regional_${domain}_valid_${valid}_${var}_${stat}_${lead}.png  evs.href.${stat}.${var}_${level}.last${past_days}days.${scoretype}_valid_${valid}.${new_lead}.${new_domain}.png
+     fi
+      
+      done #lead
+    done #domain
+   done #stat
+  done  #var
+ done   #valid 
+done    #score_type
+
+tar -cvf evs.plots.href.grid2obs.cape.past${past_days}days.v${VDATE}.tar *.png
+
+cp  evs.plots.href.grid2obs.cape.past${past_days}days.v${VDATE}.tar  $COMOUT/.  
```

Comment:
Hi @BinbinZhou-NOAA , another EE2 thing to point out. We are supposed to use the $SENDCOM variable, when set to YES, when we need to copy data to $COMOUT. Can you modify this line to be encased in an if block, like this?
```
if [ $SENDCOM="YES" ]; then
  cp evs.plots.href.grid2obs.cape.past${past_days}days.v${VDATE}.tar $COMOUT/.
fi
```
This will need to be adjusted in any scripts/ and ush/ files that copy data to $COMOUT.